src/constants/pet-stats.js

```diff
@@ -2642,16 +2667,10 @@ class Jellyfish extends Pet {
   }
 
   get first() {
-    const multHealth = getValue(this.rarity, { epic: 1 });
-    const multMana = getValue(this.rarity, { epic: 0.5 });
+    const mult_mana = getValue(this.rarity, { epic: 0.5 });
```

Comment:
If you can fix  this

> GitHub Actions
/ es-lint
Variable name `mult_mana` must match one of the following formats: camelCase, UPPER_CASE     

so `multMana`